src/web/routes/application/select-address/select-address.test.js

```diff
@@ -159,7 +200,7 @@ test('findAddress throws an error when the address is not found', (t) => {
 
   t.throws(
     findAddress.bind(null, selectedAddress, postcodeLookupResults),
-    'Unable to find selected address in list of postcode lookup results',
+    `Unable to find ${selectedAddress} in list of addresses: ${postcodeLookupResults}`,
```

Comment:
This test is incorrect as this is not the error that is thrown. I think this test is passing because this argument should be a regex

Reply:
Updated to use a regex.